oak_ml_transparency/mnist/README.md

```diff
@@ -135,3 +135,22 @@ following command to upload the tar file:
 
 The command returns the digest of the file, which you can use for downloading
 the file from Ent.
+
+## Pushing to container registry
+
+1. Build the Docker image and tag it with
+   `europe-west2-docker.pkg.dev/oak-ci/mnist-eval/mnist-eval`:
+
+```bash
+docker build -t europe-west2-docker.pkg.dev/oak-ci/mnist-eval/mnist-eval:latest .
+```
```

Comment:
Indentation is incorrect. This interrupts the list (in fact you can see in the render that the next item is also numbered as 1 again, while it should be rendered as 2)